### PR TITLE
Readme updates 

### PR DESCRIPTION
**Pull request by jnunyez**

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+
+This repository contains python modules for analyzing and generating synchronization test results by means of consuming input data coming from:
+
+* ptp4l, ts2phc, and phc2sys logs coming from the PTP operator
+
+* collector files 
+
+These are the main python modules:
+
+* demux: Demultiplex log messages from a single multiplexed source. Demultiplex log messages for the specified parser from the multiplexed content in input. For each demultiplexed log message print the canonical data produced by the parser as JSON.
```
Comment (nocturnalastro) [inline README.adoc:11]: Log messages doesn't feel right here but I can't think of phrase ATM

File: doc/reference_config.md [hunk 2]
```diff
@@ -1,72 +0,0 @@
-# Configuration File
```
[the remark this answers is not in the document]
Comment (jnunyez) [reply doc/reference_config.md:1]: I think we are not using it and we have not followed that format, can confuse contributor. But we probably need to have something like this with other tags in vse-sync-collection-tools.

[the remark this answers is not in the document]
Comment (nocturnalastro) [reply doc/reference_config.md:1]: We do have configuration files in the implementations though right?

And yes I agree when we add config files into the collection tool we will need to add a doc like this.

**Review by crwr45 — APPROVED** [review]
The commits would be better squashed or reworded but otherwise this looks OK to me.

File: doc/reference_config.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (jnunyez) [reply doc/reference_config.md:1]: That is correct. What we can do is to add a `config.yaml` similar to the one used in reference implementation to be used in analyze module.

File: README.adoc [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+
+This repository contains python modules for analyzing and generating synchronization test results by means of consuming input data coming from:
+
+* ptp4l, ts2phc, and phc2sys logs coming from PTP operator logs
+
+* collector files generated by collector tools developed in https://github.com/redhat-partner-solutions/vse-sync-collection-tools[collectors]
+
+These are the main python modules:
+
+* link:src/vse_sync_pp/demux.py[demux]: Demultiplex gathered data messages of a given identifier coming from a single multiplexed data source. Data source is a collector file. For each demultiplexed data message of a given identifier print the canonical data produced by the parser as JSON using `parse` module.
```
Comment (natifridman) [inline README.adoc:11]: Not sure I understand when should I use `demux` or `parse`

File: doc/CONTRIBUTING.adoc [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+== Analyzers
+
+Log file analyzers are contributed into `src/vse_sync_pp/analyzers/`.
+
+Each log file analyzer myst be represented by a separate class that:
+
+* is provided in an appropriate module
+* is registered in `vse_sync_pp.analyze.ANALYZERS`
+* has unit test cases built by class `AnalyzerTestBuilder`
+    * (`tests/vse_sync_pp/analyzers/test_analyzer.py`)
```
Comment (natifridman) [inline doc/CONTRIBUTING.adoc:32]: Derived `TimeErrorAnalyzer` classes must override class attribute `locked`, specifying a frozenset of values representing locked states.

Also, I think we should explain what `locked` means

File: README.adoc (hunk 3 above)
Comment (jnunyez) [reply README.adoc]: Improved text a bit.

File: doc/CONTRIBUTING.adoc (hunk 4 above)
Comment (jnunyez) [reply doc/CONTRIBUTING.adoc:33]: Added it in the contributor guide as example.

**Review by natifridman — APPROVED** [review]

**Pull request merged**